src/widget/tooltipqopengl.cpp

```diff
@@ -7,14 +7,18 @@
 #include <QToolTip>
 #include <memory>
 
-ToolTipQOpenGL::ToolTipQOpenGL() {
+#include "widget/wglwidget.h"
+
+ToolTipQOpenGL::ToolTipQOpenGL()
+        : m_active(true),
+          m_pWidget(nullptr) {
```

Comment:
the tooltip is active without a widget? That doesn't seem right.

Reply:
The actual value does not matter because it is adjusted during MixxxMainWindow::initialize();

Reply:
Ah I misinterpreted the meaning of the variable nevermind.